src/summaries/summaries-nav.component.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect } from "react";
+import PatientChartOverview from "../summaries/overview/patient-chart-overview.component";
+import { levelTwoRoutes } from "../chart-review/level-two-routes.component";
+import { Breadcrumbs } from "../breadcrumbs/breadcrumbs.component";
+import { Route, Link, Redirect, useHistory, useParams } from "react-router-dom";
+import styles from "./summaries-nav.css";
+import { RouteComponentProps } from "react-router";
+
+export default function SummariesNav(props: any) {
+  let { patientUuid } = useParams();
+  const history = useHistory();
+
+  const navItems = [
+    {
+      name: "Overview",
+      path: `/patient/${patientUuid}/chart/summaries/overview`
+    },
+
+    {
+      name: "HIV",
+      path: `/patient/${patientUuid}/chart/summaries/hiv`
+    }
+  ];
+
+  let i = 0;
+  navItems.forEach((item, index) => {
+    if (item.path === history.location.pathname) {
+      i = index;
+    }
+  });
+
+  const hasPath = item => item.path.indexOf(history.location.pathname) > -1;
+  const [selected, setSelected] = React.useState(navItems.findIndex(hasPath));
+
+  function Hiv(props: any) {
+    return <div>Hi</div>;
+  }
+
+  /*
+  React.useEffect(() => {
+    navItems.map((item, index) => {
+      if (item.path === history.location.pathname) {
+        setSelected(index);
+      }
+    });
+  }, []);
+  */
```

Comment:
We should consider removing any unused/commented code.

Reply:
gone